freebasics/urls.py

```diff
@@ -57,6 +59,9 @@
         include('molo.polls.urls',
                 namespace='molo.polls',
                 app_name='molo.polls')),
+    url(r'^403/$', TemplateView.as_view(template_name='403.html')),
+    url(r'^404/$', TemplateView.as_view(template_name='404.html')),
+    url(r'^500/$', TemplateView.as_view(template_name='500.html')),
```

Comment:
@miltontony Yeah - It was for testing. :) I think it's helpful to have an easy way to look at them.